linux_docker_resources/Dockerfile-RHEL

```diff
@@ -42,28 +42,33 @@ RUN dnf update \
 # Initialize rosdep
 RUN rosdep init && rosdep update
 
-# Install some stuff via Pip which we don't have available in EPEL
-RUN python3 -m pip install pydocstyle
-
 # Install a newer version of pytest
 RUN python3 -m pip install -U pytest pytest-rerunfailures
 
 # These dependencies were generated by invoking rosdep on the latest version of the target workspace:
+# rosdep install --from-paths src --ignore-src --simulate --reinstall --skip-keys ...
 RUN dnf install \
     CUnit-devel \
     acl \
     assimp-devel \
-    bison \
```

Comment:
I'm fairly confident that `bison` is required to build CycloneDDS by default, though that dependency is *not* properly expressed in https://github.com/eclipse-cyclonedds/cyclonedds/blob/releases/0.9.x/package.xml .  I'm surprised the build succeeded without that, maybe it got pulled in by something else?

Reply:
As of March, it is optional: https://github.com/eclipse-cyclonedds/cyclonedds/pull/1155

Maybe it would be best to leave it in for Galactic, though. I'll revert it.